ci_framework/roles/artifacts/tasks/cluster_info.yml

```diff
@@ -78,3 +78,13 @@
         chdir: "{{ cifmw_artifacts_basedir }}/logs/edpm"
       changed_when: true
       ignore_errors: true
+
+    - name: Dump cluster info for debugging
+      ansible.builtin.shell: |
+        mkdir cluster_info
+        oc cluster-info dump --all-namespaces -o yaml --output-directory={{ cifmw_artifacts_basedir }}/logs/cluster_info
+        oc cluster-info -o yaml > {{ cifmw_artifacts_basedir }}/logs/cluster_info.yaml
```

Comment:
since you use the `chdir` parameter, there's no need to put the full path for the output-direcory, nor the shell redirect... ?